editor.planx.uk/src/pages/FlowEditor/components/Settings/shared/SettingsForm.tsx

```diff
@@ -0,0 +1,65 @@
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import { FormikProps } from "formik";
+import React from "react";
+import EditorRow from "ui/editor/EditorRow";
+import InputGroup from "ui/editor/InputGroup";
+import InputLegend from "ui/editor/InputLegend";
+import ErrorWrapper from "ui/shared/ErrorWrapper";
+
+type SettingsFormProps<TFormikValues> = {
```

Comment:
Here's the core difference in this PR.

The previous props took a hardcoded `TeamTheme`, we're now using a [generic type](https://www.typescriptlang.org/docs/handbook/2/generics.html) here.